kirovy/models/cnc_user.py

```diff
@@ -0,0 +1,65 @@
+import uuid
+
+from django.contrib.auth.models import AbstractBaseUser
+from django.db import models
+from django.utils.translation import gettext as _
+from kirovy import typing as t
+
+from kirovy.objects import CncnetUserInfo
+
+__all__ = ["CncUser"]
+
+
+class CncUser(AbstractBaseUser):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    cnc_net_id = models.IntegerField(
+        unique=True,
+        editable=False,
+        help_text=_("The user ID from the CNCNet ladder API."),
+    )
+
+    username = models.CharField(
+        null=True, help_text=_("The name from the CNCNet ladder API.")
+    )
+    """:attr: The username for debugging purposes. Don't rely on this field for much else."""
+
+    verified_map_uploader = models.BooleanField(null=False, default=False)
+    """:attr: This user is a map uploader verified by cncnet staff."""
```

Comment:
What would this affect?

I think we ought to have roles/permissions model down the road. Not sure if this needs to be in the first iterations but in future this should be a permission I think.

Reply:
This was intended to be a "known uploader" type field. Like for people who we know are good at map making and are active in the community. Kind of like a Twitter verification saying "this person is who they say they are"

Reply:
Hmm I see, I thought it would affect some permissions. nvm then